src/views/AddPriceMultiple.vue

```diff
@@ -246,9 +246,26 @@
               ></v-text-field>
             </v-col>
           </v-row>
-          <div class="d-inline">
+          <div class="d-flex">
             <v-checkbox v-model="productPriceForm.price_is_discounted" :label="$t('AddPriceSingle.PriceDetails.Discount')" hide-details="auto"></v-checkbox>
+            <v-checkbox v-model="isChangeCurrency" :label="$t('AddPriceSingle.PriceDetails.ChangeCurrency')" hide-details="auto"></v-checkbox>
```

Comment:
i feel it's misleading here to have a checkbox in a form, that is for a field not related to the form.

Reply:
alternative would be : 
- if the user has 1 currency set in its settings, then keep the current behavior, and add a sentence / info popup to explain how to change / add currency
- if the user has multiple curencies, then allow choosing with a dropdown